apps/calendar/calendar.js

```diff
@@ -281,7 +282,11 @@ const showMenu = function() {
     },
     /*LANG*/"Exit": () => load(),
     /*LANG*/"Settings": () => {
-      const appSettings = eval(require('Storage').read('calendar.settings.js'));
+      const appSettings = () => eval(require('Storage').read('calendar.settings.js'))(() => {
+        loadSettings();
+        loadEvents();
+        showMenu();
+      });
       appSettings(() => {
```

Comment:
The function passed here as an argument is unused - was there a different change intended here?